Automattic/wp-calypso
# Calypso Build Package: Add `calypso-build` bin shorthand

#### Changes proposed in this Pull Request

- [x] Alias `bin/build-packages` as `transpile` bin. This will allow us to add more bin scripts.
- [x] Add `calypso-build` alias for `webpack --config=webpack.config.js`

Relevant docs: https://docs.npmjs.com/files/package.json#bin
#### Testing instructions

```
npm run distclean && npm ci
```

Verify that `npm run build-packages` still transpiles packages code and writes it to `packages/*/dist/cjs` and `packages/*/dist/esm`, respectively.

```
npx lerna run build
```

Verify that `apps/` are built successfully.
